# HG changeset patch
# User Rik <rik@octave.org>
# Date 1383500935 28800
#      Sun Nov 03 09:48:55 2013 -0800
# Node ID 18096cd30339c0626f3c46b09d8270e084f576b3
# Parent  2ff47214de72dc201b50670bcff186405e10e285
maint: Update gnulib-hg repo to current gnulib tip ahead of 3.8 release.

* .hgsubstate: Update gnulib-hg sub-repo pointer.

diff --git a/.hgsubstate b/.hgsubstate
--- a/.hgsubstate
+++ b/.hgsubstate
@@ -1,1 +1,1 @@
-6057744acd2c71c069a4b171c5fe1ff0d86c9e5f gnulib-hg
+6d4e36653a40da6604507406f2a97e3e64bf9dbf gnulib-hg
